### DISCUSSION TITLE
Docs generating incorrectly for Post Request with Non-Pydantic Body

### DISCUSSION THREAD

**Discussion by aiwhoo** [fastapi/fastapi#8416] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI, Request, Depends, Body
import uvicorn
from fastapi.middleware.cors import CORSMiddleware
import time
import database

tags_metadata = [
    {
        "name": "Recording",
        "description": "Utilities for session setup, reset, clocking, and synching multiple computers",
    },
    {
        "name": "Reporting",
        "description": "Graphing, Summarizing Data, and Submitting to Firebase",

    },
]
app = FastAPI(openapi_tags=tags_metadata)
app.add_middleware(
    CORSMiddleware,
    allow_origins=["*"],
    allow_credentials=True,
    allow_methods=["*"],
    allow_headers=["*"],
)

SESSION_ID_EXAMPLE = "AI18ASM"
COMPUTER_ID_EXAMPLE = "Gate 1"
TIME_EXAMPLE = "2018-08-21 10:31:53.3840302"

#reset session
@app.post("/reset_session/", tags=["Recording"])
async def reset_session( session_id: str = Body(example=SESSION_ID_EXAMPLE, title="Session title bob", description="bob the builder")):
    return database.reset_session(session_id)

#reset session/computer_id
@app.post("/reset_session_computer/", tags=["Recording"])
async def reset_session_computer(session_id : str = Body(example=SESSION_ID_EXAMPLE, embed=True, title="sessions are happy"),computer_id : str = Body(example=COMPUTER_ID_EXAMPLE, description="computer description", embed=True) ):
    return database.reset_session_for_computer(session_id, computer_id)
```

### Description

Open the browser and go to /docs. 

The reset_session end point has the docs auto generated correctly.

The reset_session_computer's examples are ```
{
  "session_id": "string",
  "computer_id": "string"
}
``` rather than the specified example in the function.



### Operating System

macOS

### Operating System Details

_No response_

### FastAPI Version

0.85.0

### Python Version

Python 3.9.1

### Additional Context

_No response_

**Comment by iudeen** [comment]
Why don't you use the recommended Pydantic way?

**Comment by aiwhoo** [comment]
Well, The underlying cache/limiter library which we're using requires a starlette Request object.

So even the headers would need to just be request : Request, but I still don't know how to enforce the docs to be updated with the request object when it's known that it should only have 2 params to the function.

Even so, the issue is that everything in the docs generated correctly except for the examples.

**Comment by iudeen** [comment]
I'll check this. Give me a few hours.

**Comment by aiwhoo** [comment]
Hi @iudeen any updates?
